# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/freetype1/gcc33-support.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN -x '*~' freetype-1.3.1/test/ftdump.c freetype-1.3.1-gcc-3.3.1/test/ftdump.c
--- freetype-1.3.1/test/ftdump.c	1999-09-04 09:11:29.000000000 +0200
+++ freetype-1.3.1-gcc-3.3.1/test/ftdump.c	2003-08-14 05:13:41.000000000 +0200
@@ -140,7 +140,7 @@
     old_memory += *var;
   }
 
-#define FOOTPRINT( field )  Save_Memory( &memory_footprint.##field )
+#define FOOTPRINT( field )  Save_Memory( &memory_footprint.field )
 
 
   static void
@@ -153,7 +153,7 @@
   }
 
 #define PRINT_MEM( field, string ) \
-          Print_Mem( memory_footprint.##field, string )
+          Print_Mem( memory_footprint.field, string )
 
 
   /* Print the memory footprint */
